solidity-verifiers/src/evm.rs

```diff
@@ -71,10 +71,27 @@ pub fn compile_solidity(solidity: impl AsRef<[u8]>, contract_name: &str) -> Vec<
 /// `contract_name` is provided since `solc` may compile multiple contracts or libraries.
 /// hence, we need to find the correct binary.
 fn find_binary(stdout: &str, contract_name: &str) -> Option<Vec<u8>> {
-    let start_contract = stdout.find(contract_name)?;
-    let stdout_contract = &stdout[start_contract..];
-    let start = stdout_contract.find("Binary:")? + 8;
-    Some(hex::decode(&stdout_contract[start..stdout_contract.len() - 1]).unwrap())
+    let intro_str = format!("======= <stdin>:{contract_name} =======\nBinary:\n");
+
+    println!("intro_str: {:?}", intro_str);
+    println!("stdout: {:?}", stdout);
```

Comment:
Could we have these `println` calls disabled in release mode? It is okay to have these information when debugging, but end users may find it confusing when using the library.

Ideally, maybe we can have a feature (for example, `print-trace` just as `ark-std/print-trace`) that controls all debug information include this. But simply hiding `println`s behind something like `#[cfg(debug_assertions)]` is also fine with me.

Reply:
my best apologies - these were just leftovers from my debugging... I've removed them